src/Simulation/TargetDefinitions/Decompositions/ApplyUncontrolledTAdj.qs

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Intrinsic {
+    open Microsoft.Quantum.Math;
+
+    /// # Summary
+    /// Applies the -π/8 gate to a single qubit. Note that the Controlled functor is
+    /// not supported.
+    ///
+    /// # Description
+    /// \begin{align}
+    ///     TAdj \mathrel{:=}
+    ///     \begin{bmatrix}
+    ///         1 & 0 \\\\
+    ///         0 & e^{i -\pi / 4}
+    ///     \end{bmatrix}.
+    /// \end{align}
+    ///
+    /// # Input
+    /// ## qubit
+    /// Qubit to which the gate should be applied.
+    internal operation ApplyUncontrolledTAdj (qubit : Qubit) : Unit {
+        ApplyUncontrolledRz(-PI() / 4.0, qubit);
+    }
+}
```

Comment:
(Can be a false alarm)
When I look at the implementation of the subroutines `ApplyUncontrolledT[Adj]()` I see the correspondence between the code and the documentation, e.g. the code fragment `PI() / 4.0` corresponds to the doc-comment fragment `\pi / 4`.

But when I look at the implementation of the subroutines `ApplyUncontrolledS[Adj]()` (earlier), I don't see such a correspondence. In particular, I see the code fragment `PI() / 2.0`, but I don't see the corresponding doc-comment fragment `\pi / 2`.

Reply:
This is expected, as the S gate is equivalent to TT (running T twice) and so the entry in the matrix is just imaginary `i` as seen in the description of the S gate (or phase gate or sqrt(Z) gate) in https://en.wikipedia.org/wiki/List_of_quantum_logic_gates#Clifford_qubit_gates.